controllers/front/AuthController.php

```diff
@@ -144,6 +144,8 @@ public function initContent()
 
         // Call a hook to display more information on form
         $this->context->smarty->assign(array(
+        	'HOOK_AUTHENTICATE_FORM_BOTTOM' => Hook::exec('displayAuthenticateFormBottom'),
```

Comment:
Hi,

Thank you for your PR. Are you using tabs or spaces for indentation ?

Regards